### ISSUE TITLE
Using subpopulations and other attributes from the plans file

### ISSUE THREAD

**Issue by nol-ram** [matsim-org/matsim-code-examples#676]
Hello dear MATSim-Community,

We are currently working on getting subpopulations to work in the “Ruhrgebiet”-Scenario (v.1.1, MATSim 12). Unfortunately adding the subpopulation tag in the agents plans did not result in subpopulations being part of the agent objects, calling PopulationUtils.getSubpopulation(person) returned null. The same applies for any other arbitrary attribute we tried setting in the plans.xml file.

We are not sure why this is the case, as the user guide lacks further information if additional steps are necessary. Do we need to define the different subpopulations somewhere else first or does the Ruhrgebiet scenario have a non-standard scenario loading mechanism?

Consequently, we tried a “dirty” workaround by prefixing agent ids with a string to then use “PopulationUtils.putSubpopulation( person, prefix )” in the setup process. Here another problem occurs: The string is getting lost while reading the population file.
Tl:dr: How to get subpopulations in MATSim 12 to work?
